doc/source/all-styles.rst

```diff
@@ -0,0 +1,48 @@
+
+Style
+=====
+
+In the PyAnsys ecosystem, three key styles—packaging, coding, and documentation—
+serve as foundational pillars to enhance the overall developer experience.
+
+- Packaging style focuses on creating clear, open source APIs hosted on GitHub,allowing
+  for reusable packages that can be updated independently of the Ansys release schedule.
+
+- Coding style adheres to PEP 8 and aligns with the conventions of major data science libraries
+  like NumPy, SciPy, and pandas, ensuring consistency and readability.
+
+- Documentation style emphasizes the significance of well-documented APIs, 
+  offering increased adoption, an improved on-boarding experiences, and streamlined code maintenance.
+
+All PyAnsys libraries follow the Google developer documentation style guide, 
```

Comment:
Make ``Google developer documentation style guide`` a link: https://developers.google.com/style/

Reply:
Did you mean the space for the second line of bullet point? 
?